cparser/libFirm doesn't understand __only_inline.
This lets us work around it.

Index: ir/be/platform.c
--- ir/be/platform.c.orig
+++ ir/be/platform.c
@@ -179,6 +179,7 @@ void ir_platform_set(ir_machine_triple_t const *machin
 		goto BSD;
 	} else if (strstart(os, "openbsd")) {
 		ppdef1("__OpenBSD__");
+		ppdef("_ANSI_LIBRARY", ""); /* avoid __only_inline */
 		goto BSD;
 	} else if (strstr(os, "bsd") != NULL) {
 BSD:
